### DISCUSSION TITLE
Unable to boot from latest 4.2 beta-dev release

### DISCUSSION THREAD

**Discussion by delston** [OnionUI/Onion#876] [Q&A]
### Checklist

- [X] I am reporting a bug in the Onion OS (Note that Onion devs can’t fix inherited issues with RA Cores)
- [X] I have reviewed the [Onion Wiki](https://github.com/OnionUI/Onion/wiki) and did not find the answer
- [X] I have searched the existing (open and closed) issues to ensure I do not log a duplicate

### Onions OS Version

4.2

### Miyoo Firmware Version

202303292051

### Provide a clear and concise description of the issue

Unable to boot from the latest 4.2 beta-dev release

After copying it to the SD card, it seems to boot to the default OS.  Not sure what's going on beyond that.

### Steps to Reproduce

1. Download 4.2 pre-release for Miyoo Plus
2. Copy files to 256GB Samsung Pro Plus Micro SD
3. Install card in Miyoo plus
4. Turn on
5. Boots to stock OS

### Provide any additional context or information that may help us investigate your issue

 256GB Samsung Pro Plus Micro SD

**Comment by Aemiii91** [comment]
How did you format the card? Does it say unmounted when you go to Settings -> Device Info?

**Reply by delston** [reply]
So it is installing right now, just tried again.  Formatted with FAT32 using Refus for large SD cards. SD Card showing 0G/238G, doesn't say unmounted. 

It did start the installation process, then it rebooted and now at default OS.

**Reply by delston** [reply]
Yea, I just did, same issue

**Reply by delston** [reply]
@Aemiii91  any ideas?  Plan to buy you a coffee if you can help me out.  Pretty pumped to try Onion 4.2 on the plus.

**Reply by Aemiii91** [reply]
I'm gonna see if I can reproduce the issue in a moment. I'm not sure which step you're missing, but I know a bunch of other users have installed successfully. You could also try:
- reformatting the card
- another card

**Reply by Aemiii91** [reply]
Though, the card should be fine, since the installation went through 🤷‍♂️

**Reply by delston** [reply]
What else could it be? Bios?

**Reply by Aemiii91** [reply]
No, BIOS files are only needed for emulating (some) systems.

**Comment by Aemiii91** [comment]
Seems there's a user on Reddit with a similar issue: https://www.reddit.com/r/MiyooMini/comments/13zgz5t/upgrading_to_onion_os_not_appearing_to_work/
(No apparent solution yet)

**Reply by delston** [reply]
I've figured out what happened.  I was trying to use a 256GB memory card, for some reason the installation didn't fully go through.  I switched to a 32GB card and did a basic Fat32 format and everything worked as expected.  @Aemiii91

**Reply by Aemiii91** [reply]
That shouldn't be a problem though. Did you then copy the installed system onto the 256 card?

**Reply by delston** [reply]
Tried multiple times, 256gb card failed using fat32 large size.  32gb worked fine.

**Reply by Aemiii91** [reply]
I would still recommend you try another (large) card. It should really work fine up to 2 TB (only 1 TB exists though)

**Reply by delston** [reply]
It appears to work fine when copied to the memory card.  For some reason, when taking the installation route with the larger card it didn't work.
